src/components/dashboard/main-content/CourseRecommendations.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from 'react';
+import { Button } from '@edx/paragon';
+import { Link } from 'react-router-dom';
+import { AppContext } from '@edx/frontend-platform/react';
+
+import SkillsQuizImage from '../../../assets/images/skills-quiz/skills-quiz.png';
+
+const CourseRecommendations = () => {
+  const {
+    enterpriseConfig: {
+      slug,
+    },
+  } = useContext(AppContext);
+
+  return (
+    <div className="course-recommendations">
+      <h2 className="course-recommendations-title">Get course recommendations for you</h2>
```

Comment:
A full stop is missing at the end.

Reply:
In the ticket the text heading is `Get courses recommended for you.`

Reply:
Done